mozilla/bedrock
# [fix bug 1393132] Remove old /firefox/new/ and /firefox/firstrun/ templates

## Description
- Removes legacy `/new` and `/firstrun` templates, reorganizing & tidying up both directories.

## Issue / Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1393132

## Testing
- Run with a clean `/static` cache via `gulp static:clean`
- Make sure both `/new` and `/firstrun` pages load and work as expected.
- Make sure the copy A/B test on `/firstrun` still works as expected.
- Check to make sure I didn't delete anyhting that was still needed, or miss something that can be removed?

Demo:
- https://bedrock-demo-agibson.us-west.moz.works/en-US/firefox/new/
- https://bedrock-demo-agibson.us-west.moz.works/en-US/firefox/55.0/firstrun/
- https://bedrock-demo-agibson.us-west.moz.works/en-US/firefox/55.0/firstrun/?v=a
- https://bedrock-demo-agibson.us-west.moz.works/en-US/firefox/55.0/firstrun/?v=b
